docs/usage/self-hosted-configuration.md

```diff
@@ -652,6 +652,49 @@ By default, Renovate does not autodiscover repositories that are mirrors.
 
 Change this setting to `true` to include repositories that are mirrors as Renovate targets.
 
+## inheritConfig
+
+When this option is enabled, Renovate will look for a file `org-inherited-config.json5` in the `<orgName>/renovate-config` repo before processing a repository, and read this in as config.
+If the repository is in a nested org/group on a supported platform such as GitLab, such as `topGroup/nestedGroup/projectName` then Renovate will look in `topGroup/nestedGroup/renovate-config`.
+
+If `inheritConfig` is `true` but the inherited config file does not exist then Renovate will proceed without warning.
+If the file exists but cannot be parsed, then a config warning issue will be raised and the job aborted.
+
+Inherited config can include all valid repo config and additionally the following config options:
+
+- `bbUseDevelopmentBranch`
+- `onboarding`
+- `onboardingBranch`
+- `onboardingCommitMessage`
+- `onboardingConfig`
+- `onboardingConfigFileName`
+- `onboardingNoDeps`
+- `onboardingPrTitle`
+- `onboardingRebaseCheckbox`
+- `requireConfig`
+
+This means that orgs can change/control default behavior for whether configs are required and how repositories are onboarded.
+
+This setting is not enabled in the Mend Renovate App because each 404 from the GitHub API would count as a used API call, which would add millions of wasted API calls per week.
```

Comment:
```suggestion
We disabled `inheritConfig` in the Mend Renovate App to avoid millions of wasted API calls per week.
This is because each `404` from the GitHub API counts as a used API call.
```

I think it's nice to explain the key point in the first sentence, and give more details in the second sentence.